src/GI.h

```diff
@@ -163,18 +138,92 @@ template <class Ptcl> class GI : public Problem<Ptcl>{
                 for(int i=0; i<nptclimp; i++){
                     Ptcl ith;
                     ith.readAscii(impFile);
-                    ith.vel.x += (-1) * impVel;
+
+		    // This needs to be updated -- the code should determine the impactor and target sizes
```

Comment:
Is this comment still up to date? The code does seem to take into account `impRadi` and `tarRadi`below. Or do you mean something else with the comment? If so please expand the comment.